scripts/tuple_tree_generator/tuple_tree_generator/templates/tuple_tree_gen.py.tpl

```diff
@@ -62,9 +77,12 @@ class 'struct.name'(
     'struct.inherits.name',
     ##- endif -##
     ##- if struct.abstract -##
-    AbstractStructBase
+    AbstractStructBase,
     ##- else -##
-    StructBase
+    StructBase,
+    ##- endif -##
+    ##- if schema.base_namespace == 'model' -##
```

Comment:
So this fails to compile, I'm assuming because the same template is used for creating the `pipeline tupletree` and the `mixins` are not defined there. Please advise how to proceed.

Reply:
Make it so `get_mixin_imports` and `get_mixins` are a no-op if `--mixins` is not passed